package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@BuildHero/watermelondb",
   "description": "Build powerful React Native and React web apps that scale from hundreds to tens of thousands of records and remain fast",
-  "version": "0.20.2",
+  "version": "0.20.3",
```

Comment:
I pushed already a 0.20.3 package.
Do you wanna take over and I push the 0.20.4?
https://github.com/BuildHero/WatermelonDB/pull/4

Reply:
Sounds good!